test_file_name

```diff
@@ -0,0 +1 @@
+content-1content-2
```

Comment:
What's this file?

Reply:
Woops, that was a test artefact- removed file and updated tests, thank you!

Reply:
I still see the file here

Reply:
Oh the test didn't quite get corrected - it regenerated the file. Updated the test and removed file again. Thank you!